[Compute/CSPM Maxwell Update 1] Release Notes

divyabhushan-okta opened the pull request.

Commit:
[Compute/CSPM Maxwell Update 1]

Commit:
[Compute RN]: Month name correction

Pubs-MV commented on cspm/rn/prisma-cloud-compute-release-information/prisma-cloud-compute-known-issues.adoc:
```diff
@@ -15,6 +15,19 @@ The list of addressed issues are not cumulative; only the issues that are fixed
+|Fixed incorrect CVE matching to the base layer for the binaries installed without a package manager.
+There are differences in the results between an image created by Dockerfile and an image pulled by the registry. The vulnerabilities scan results attribute correctly to the base layer for the images built from a Dockerfile.
+
+//CWP-47549
```
@rcaballeromx should this be an enhancement or a fixed issue?

Please review and update the PR

rcaballeromx replied:
I moved this item to the Agentless scanning enhancements and added the blurb.

Commit:
Add .NET Core 3.1 deprecation blurb

rcaballeromx commented:
Added the .NET Core 3.1 end of support notice.

Pubs-MV committed:
Update features-introduced-in-compute-may-2023.adoc

added build .152

Pubs-MV committed:
Update prisma-cloud-compute-known-issues.adoc

deleted CWP-44527- as it was fixed in Lagrange.

removed placeholder for CWP-47549 as this was added as a feature

Pubs-MV committed:
Update features-introduced-in-compute-may-2023.adoc

edited CWP-48360

Pubs-MV committed:
Update prisma-cloud-compute-release-information.adoc
